frontend/app/src/App.tsx

```diff
@@ -1903,7 +1919,7 @@ export class App extends PureComponent<Props, State> {
   }
 
   requestFileURLs = (requestId: string, files: File[]): void => {
-    if (this.isServerConnected()) {
+    if (this.isServerConnected() && this.sessionInfo.isSet) {
```

Comment:
nit: maybe we log to console a warning that the server is not connected or the session info is not set.